code/utils/subxt-exports/src/generated/composable_dali_dev.rs

```diff
@@ -1541,10 +1541,10 @@ pub mod api {
             use super::runtime_types;
             pub struct ConstantsApi;
             impl ConstantsApi {
-                #[doc = " A fee mulitplier for `Operational` extrinsics to compute \"virtual tip\" to boost their"]
+                #[doc = " A fee multiplier for `Operational` extrinsics to compute \"virtual tip\" to boost their"]
                 #[doc = " `priority`"]
                 #[doc = ""]
-                #[doc = " This value is multipled by the `final_fee` to obtain a \"virtual tip\" that is later"]
+                #[doc = " This value is multiplied by the `final_fee` to obtain a \"virtual tip\" that is later"]
```

Comment:
likely `generated` must be excluded from cspell all togethere - like composable-subxt generated for example

Reply:
done